sycl/unittests/CMakeLists.txt

```diff
@@ -9,10 +9,16 @@ function(add_sycl_unittest test_dirname)
   # Enable exception handling for these unit tests
   set(LLVM_REQUIRES_EH 1)
 
+  if (CMAKE_BUILD_TYPE MATCHES "Debug")
+    set(sycl_lib "sycld")
+  else()
+    set(sycl_lib "sycl")
+  endif()
+
```

Comment:
What is the purpose of this change? Why can't we always use "sycl"?

Reply:
Debug tests should link with debug SYCL runtime on Windows, because Microsoft C and C++ runtime are have different ABI in for debug and release.

I made this change because SYCL unittests failed to compile on Windows in Debug build. I can make a separate PR with this change.

Anyway, I forgot to guard this change with `if (MSVC)`, so please don't merge this PR yet.